chrome-extension/content.js

```diff
@@ -189,14 +198,59 @@ async function getCategoryDropdown(userId) {
   return categoryDropdown;
 }
 
+/**
+ * Returns select HTML element of places search result options
+ * @param {String} text 
+ */
+async function getPlacesSelection(text) {
+  const searchResults = await placesSearch(text);
+  if (searchResults.length > 0) {
```

Comment:
In the catch block you return null if there is an error so you should do if (!searchResults || searchResults.length > 0)